GOVERNANCE.md

```diff
@@ -12,6 +12,11 @@ resolved.
 
 # Roles And Responsibilities
 
+## The Project
+
+The Zarr project consists of the core Zarr specification and the various
+implementations of Zarr in different programming languages.
```

Comment:
Generally agree with this sentiment, but the governance can probably only cover items that are under https://github.com/zarr-developers, no?

Reply:
I would say we should invite implementations to join the zarr-developers org. And we can add a special provision for implementations that have to live outside the org (e.g. NetCDF).

Reply:
ok, and I'm certainly fully in support, but those outside are the majority (in fact, `N-1`) Julia for example definitively decided on https://github.com/JuliaIO/Zarr.jl/ with an existing offer of https://github.com/zarr-developers.